includes/learning-mode/class-sensei-learning-mode-compat.php

```diff
@@ -151,3 +151,12 @@ public function theme_mod_custom_logo( $custom_logo ) {
 
 	}
 }
+
+/**
+ * Class Sensei_Course_Theme_Compat
+ *
+ * @ignore only for backward compatibility.
+ * @since 4.0.2
+ */
+class Sensei_Course_Theme_Compat extends Sensei_Learning_Mode_Compat {
```

Comment:
Should we add a `@deprecated` tag? Also for any aliased method mentioned above, for those we can add `_deprecated_function` calls too.

Reply:
Yeah that makes sense.